src/notifications/notifications.service.ts

```diff
@@ -0,0 +1,104 @@
+import { Injectable } from '@nestjs/common';
+import { getMessaging } from 'firebase-admin/messaging';
+import { firebaseConfig } from 'src/configs/firebase/firebase.config';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { UpdateNotificationDto } from './dto/update-notification.dto';
+import { NotificationDto } from './dto/notification.dto';
+import { User } from '@prisma/client';
+import * as firebase from 'firebase-admin';
+import * as path from 'path';
+
+firebase.initializeApp({
+  credential: firebase.credential.cert(
+    path.join(__dirname, '..', 'configs/firebase/firebase-admin.json'),
+  ),
+});
+
+@Injectable()
+export class NotificationsService {
+  constructor(private readonly prismaService: PrismaService) {}
+
+  acceptPushNotification = async (
+    user: User,
+    notification_dto: NotificationDto,
+  ) => {
+    await this.prismaService.notificationToken.updateMany({
+      where: { userId: user.id },
+      data: {
+        status: 'INACTIVE',
+      },
+    });
```

Comment:
There's no need to make other token inactive, as user may receive a notification on multiple devices at once